assets/js/sheetsAPI.js

```diff
@@ -91,7 +91,7 @@ function rawDataToTableData(rawBreadLedgerData) {
             size,
             quantity: parseInt(quantity),
             description,
-            cost: cost.split("$")[1],
+            cost: parseFloat(cost),
```

Comment:
I didn't think this would work.

Cost looks like
"$5.00", no?

Would you still need to use the split to get just 5.00 before you parsefloat?

Reply:
Not anymore - I changed the ledger column from 'Currency' format to 'Number'. So the data is retrieved from the sheet as a number without a '$' symbol.

This worked when I tried it locally.